docs/docs/cli-commands.md

```diff
@@ -735,6 +736,35 @@ $ /redwood-app/node_modules/.bin/dev-server
 17:21:49 api | ► http://localhost:8911/user/
 ```
 
+### generate job
+
+Generate a background job file (and optional tests) in `api/src/jobs`.
+
+```bash
+yarn redwood generate job <name>
+```
+
+| Arguments & Options  | Description                                                                           |
+| -------------------- | ------------------------------------------------------------------------------------- |
+| `name`               | Name of the job ("Job" suffix is optional)                                            |
+| `--force, -f`        | Overwrite existing files                                                              |
+| `--typescript, --ts` | Generate TypeScript files. Enabled by default if we detect your project is TypeScript |
```

Comment:
"is using TypeScript" maybe? or "is a TypeScript project"?

Reply:
I copied from one of the other ones, so we'll probably have a few more to change!

Reply:
Meh. Leave it then